src/fs/v3/IpfsBlockStore.ts

```diff
@@ -0,0 +1,62 @@
+import * as uint8arrays from "uint8arrays"
+import { IPFS } from "ipfs-core"
```

Comment:
Better to do `import type` so we don't need to add `ipfs-core` to the dependencies (unless Typescript removes this too?). Also, I've replaced this with `ipfs-core-types` everywhere, because it was creating issues with the unstable packages for ipfs-core and ipfs-core-types I decided to use (to avoid bundler problems which the currently stable versions have).